tox.ini

```diff
@@ -35,4 +35,5 @@ deps =
   -r{toxinidir}/docs/docsite/requirements.in
   -c{toxinidir}/docs/docsite/requirements.txt
 commands =
+  python {toxinidir}/docs/docsite/rst/rest_api/_swagger/download-json.py
```

Comment:
any particular reason to use a python call here instead of curl or wget? Not very familiar with tox command limitations

Reply:
hey @fosterseth thanks for the review and question. this really came down to the read the docs build moreso than tox. somewhat surprisingly neither wget or curl seemed to be available there. in an effort to make sure the build is consistent across environments the python call seemed like a good way to go.